tests/default/_core/refresh.yaml

```
@@ -0,0 +1,10 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test the _refresh endpoint to ensure basic refresh functionality.
```

Super nit. We are testing the spec, not the functionality of the product. 

```suggestion
description: Test _refresh.
```